dev-app/src/main/java/com/uid2/dev/ui/MainScreenViewModel.kt

```diff
@@ -77,18 +80,39 @@ class MainScreenViewModel(
         viewModelScope.launch {
             when (action) {
                 is MainScreenAction.EmailChanged -> {
+                    _viewState.emit(LoadingState)
+
                     try {
-                        // For Development purposes, we are required to generate the initial Identity before then
-                        // passing it onto the SDK to be managed.
-                        _viewState.emit(LoadingState)
-                        api.generateIdentity(action.address, EMAIL)?.let {
-                            manager.setIdentity(it)
+                        if (action.clientSide) {
+                            // Generate the identity via client side token generation.
```

Comment:
_Generate the identity via Client Side Integration (client side token generation)_

?